app/config/wp-demo/download.sh

```diff
@@ -4,12 +4,12 @@
 
 ###############################################################################
 
-[ -z "$CMS_VERSION" ] && CMS_VERSION=4.3
+[ -n "$CMS_VERSION" ] && CMS_VERSION=" --version=$CMS_VERSION"
```

Comment:
It may be a little nitpicky, but I care b/c `CMS_VERSION` is a document variable in https://github.com/civicrm/civicrm-buildkit/blob/master/src/civibuild.defaults.sh#L86 . Someone skimming the list of vars wouldn't expect it to include `--version=4.3`.